src/modules/auth/__test__/register.test.ts

```diff
@@ -86,9 +84,7 @@ describe('POST /api/auth/register', () => {
 
 		expect(response.statusCode).toBe(400);
 		expect(response.json()).toMatchObject({
-			error: 'Bad Request',
-			message: "body must have required property 'email'",
-			statusCode: 400,
+			_: ['Email is required'],
```

Comment:
All these required errors should belong to their respective fields.
So the response should be:
```suggestion
			email: ['Email is required'],
```

Reply:
The required error is only possible to be set in the overall object, and not on the specific field - This can be seen in the Schema file

Reply:
Mhh, annoying but I guess it will do